Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pwasm-std"
-version = "0.6.0"
+version = "0.7.0"
```

Comment:
We'd better change this package metadata it in separate pr/commit (i can handle later myself)

Reply:
I think it's ok to change them syncroniosly, because you cannot refer `alloc` package from elsewhere. So this is actually an atomic change in only place. Thus I don't see why it should be separated.

Reply:
changes in metadata better keep separated from logic (i mean for all tomls - for alloc and std itself)

Reply:
I do believe in the opposite: you can have two packages with same methadata which are completely different, and it's way worse, so I think that changes in logic should always go with different metadata, because otherwise you can break tools that determines if packages are the same by using their metadata. Separating them in different commits breaks tooling expectations.

Reply:
Version publish is not bound excusively to your changes, we might get other changes in and publish 0.4 / 0.7 versions all together ;)

Reply:
yeah, I think the practical reason to manage versions in one place becomes clear especially when you have several parallel branches.